### DISCUSSION TITLE
Welcome to Bentilzone-Restaurant Discussions!

### DISCUSSION THREAD

**Discussion by qbentil** [qbentil/Bentilzone-Restaurant#38] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by eyotomtomm** [comment]
Can u send me the catagory design code for me to add

**Reply by qbentil** [reply]
Hello @eyosii 
Kindly elaborate on what you meant so I can assist you

**Reply by eyotomtomm** [reply]
hello@qbentil 
i loved ur work and was gonna put foods on the website but couldn't find the code. I have attached the screen shot for the code i need to update or change 

![Screenshot 2023-07-11 112948](https://github.com/qbentil/Bentilzone-Restaurant/assets/85642199/c3f2e14e-6a41-4813-8853-7640ab53aa47)

**Reply by qbentil** [reply]
Hey @eyosii ,
You want to add more food items to it?
If so, I can make your account and admin account so you do that from the Admin Dashboard.

The food items you are seeing are not static. They are actually coming from firebase.

**Comment by moad-dd** [comment]
Hello @qbentil 
I hope you are doing well 
can you send me the schema of Firebase db
the schema of the DB will be useful for me to add my db

**Reply by qbentil** [reply]
@moad-dd 
## User Schema:
![image](https://github.com/user-attachments/assets/63fb97c1-ca33-4ed7-8098-7d2329d09fa7)

## Foot Schema:
![image](https://github.com/user-attachments/assets/f434b356-4ce0-45d6-88a0-e59d79e6be9b)

## Cart Item Schema:
![image](https://github.com/user-attachments/assets/003eb962-9a65-4238-a725-1780687a1a91)
